scripts/linux/runE2ETest.sh

```diff
@@ -278,6 +278,10 @@ function process_args() {
     print_highlighted_message 'Process arguments'
     saveNextArg=0
     BYPASS_EDGE_INSTALLATION=""
+    CONNECT_MANAGEMENT_URI="unix:///var/run/iotedge/mgmt.sock"
```

Comment:
Just leave empty string for default value in this script.

Reply:
Makes sense.